AENS.md

```diff
@@ -66,19 +65,18 @@ could allow us an easier path for future update to the fee structures.
 
 Every entry in the `.test` namespace pays the same amount of fees.
 
+The new mechanism planted in Lima hard fork introduces auctions.
+We propose very simplistic scheme. The claim action becomes claim attempt. It can be followed by another claim from an account different than one set in preclaim for given name. There is finite amount of time when the follow up claim is allowed. This time is expressed in height delta computed from function of length of the name. In practice it means that the shorter the name, the longer we wait for another claim attempt. This should work as bidding. Furthermore, the initial fee is also a value of the function of a name length. Each next bid has to be higher by `X` tokens. Finally, we set a protocol protected length of name that is not subject to bidding and will be claimed immediately after confirming claim transaction. All, functions, base fee, free length value and price progression may be subject to changes with governance mechanism. Non-bidding path of the name claim is purposed for development or testing. Initial non-bidding length will be 32.
```

Comment:
This is really hard to follow. And please format it to 80 characters per line

Reply:
@knarz I reformatted and re-phrased the algo